src/packages/excalidraw/index.tsx

```diff
@@ -57,6 +57,7 @@ const ExcalidrawBase = (props: ExcalidrawProps) => {
       ...canvasActions,
     },
     tools: {
+      ...props.UIOptions?.tools,
```

Comment:
it works, but its not type-safe, if its not so complicated we could assign types here

Reply:
I prefer the non typesafe way if there are less changes.